website/src/docs/aws-s3.md

```diff
@@ -102,6 +102,49 @@ strings: {
 }
 ```
 
+### `getUploadHeaders()`
+
+Set the default headers for the request to be sent to the server.
+A common use case is adding the `Authorization` header with your JWT token.
```

Comment:
We should clarify that this is only used if you do not override `getUploadParameters()`

Reply:
Would be easier if `getUploadParams.url` option was optional. Then we don't need `getUploadHeaders`.

Reply:
I'm not sure I understand. If `url` was optional, the plugin wouldn't know where to upload to.